### PR TITLE
Make copying GC the default again

### PR DESCRIPTION
**Pull request by osa1**
See #2909

### CODE REVIEW

**Comment by osa1** [comment]
@crusso please feel free to edit the changelog entry

**Review by ulan — APPROVED** [review]
Thanks, lgtm!

**Review by crusso — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -42,21 +42,12 @@
 
   If you encounter such cases, please adjust the type annotation.
 
-* The garbage collection scheduling algorithm is improved once again, and the
```
Comment (ggreif) [inline Changelog.md:45]: if you change stuff here, better remove
`## 0.6.13 (2021-11-19)` from above, or change that to `0.6.14`.

Comment (ggreif) [reply Changelog.md:45]: When releasing, I'll leave a note that `0.6.13` was pulled

Comment (osa1) [reply Changelog.md:45]: I'm offline now, everyone can push to my branch, could you do the tweaks please?

Comment (ggreif) [reply Changelog.md:45]: Done

**Review by ggreif — APPROVED** [review]

**Review by crusso — APPROVED** [review]

**Pull request merged**